src/stories/Library/slider/slider.scss

```diff
@@ -0,0 +1,75 @@
+.slider {
+  @include vertical-spacing;
+}
+
+.slider__header {
+  @include layout-container;
+
+  display: flex;
+  margin-bottom: $s-lg;
+}
+
+.slider__title {
+  @extend %title-underline;
+  @include typography($typo__h2);
+
+  display: flex;
+}
+
+.slider__controls {
+  display: none;
+  flex-direction: column;
+  flex-grow: 1;
+  align-content: end;
+
+  @include media-query__small {
+    display: flex;
+  }
+}
+
+.slider__control {
+  width: 115px;
+  max-width: 100%;
+  margin-left: auto;
+  transition: opacity 0.2s ease-in-out;
+  cursor: pointer;
+
+  &[disabled] {
+    opacity: 0;
+    width: 0;
```

Comment:
```suggestion

```
When getting to the end of either direction in the slider, the oppising arrow will "jump" before turning invisible when i test. 

I think this can be changed by just removing this width css.